Documentation/LinuxVMGuide/README.adoc

```diff
@@ -0,0 +1,57 @@
+﻿= Setting up a Linux Virtual Machine
+
+This guide assumes the host operating system is 64 bit Windows
+This guide was written for Linux Mint 18.3.
+The steps include installing the required software on the host machine and setting up Linux Mint as a virtual machine (VM)
+
+== 1. Install Virtualbox
+Download https://www.virtualbox.org/wiki/Downloads[Oracle Virtual Box]
+The latest version at writting was 5.2.4. 
+Download the VirtualBox Extension Pack from the same page.
+Install both.
+
+== 2. Setup a Virtual Machine
+Create a new virtual machine in Virtualbox using the _New_ icon.
+
+
+image::https://github.com/StudentOrganisationForAerospaceResearch/DevelopmentResources/blob/garnettanner/stm32-example/STM32Resources/BuildEnvironment_Setup/images/1.PNG[]
```

Comment:
These links are very likely to break, you should use relative paths.

You can reference the README from this project.
https://raw.githubusercontent.com/JeLLyNinjas/TerminalFighter/master/README.adoc
https://github.com/JeLLyNinjas/TerminalFighter/blob/master/README.adoc

Usually, if your name appears anywhere in a file, it's a red flag :)